src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -67,17 +67,56 @@ ucs_config_field_t uct_ib_mlx5_iface_config_table[] = {
      "+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++\n",
      ucs_offsetof(uct_ib_mlx5_iface_config_t, ar_enable), UCS_CONFIG_TYPE_TERNARY_AUTO},
 
+    {"CQE_ZIPPING_ENABLE", "no",
+     "Enable CQE zipping feature. CQE zipping reduces PCI utilization by merging several similar CQEs\n"
+     "to a single CQE written by the device.",
+     ucs_offsetof(uct_ib_mlx5_iface_config_t, cqe_zipping_enable), UCS_CONFIG_TYPE_TERNARY},
+
     {NULL}
 };
 
-ucs_status_t uct_ib_mlx5_create_cq(uct_ib_iface_t *iface, uct_ib_dir_t dir,
-                                   const uct_ib_iface_init_attr_t *init_attr,
-                                   int preferred_cpu, size_t inl)
+static UCS_F_ALWAYS_INLINE ucs_status_t
```

Comment:
1. doesn't need to be UCS_F_ALWAYS_INLINE
2. to fix build issue, the whole function should be under `#if HAVE_DECL_MLX5DV_CQ_INIT_ATTR_MASK_CQE_SIZE`